actioncable/lib/action_cable/connection/base.rb

```diff
@@ -49,7 +49,7 @@ class Base
       include Authorization
 
       attr_reader :server, :env, :subscriptions, :logger
-      delegate :worker_pool, :pubsub, to: :server
+      delegate :worker_pool, :adapter, to: :server
```

Comment:
I agree with the earlier comments that calling this `adapter` rather than `pubsub` is a step backwards. Adapter is a generic pattern. We could have adapters for all sorts of things. This is specifically an adapter that provides the pubsub backend we need. So let's stick with the `pubsub` name. Our use of `broadcast` vs `publish` is more just to stay within the cable metaphor. It's still the same thing.